app/assets/javascripts/tylium/behaviors.js.coffee

```diff
@@ -191,6 +191,10 @@ document.addEventListener "turbolinks:load", ->
   if $('[data-behavior~=restrict-height').length
     checkOverflow = ->
       $('[data-behavior~=restrict-height').each ->
+        if $(this).is('[data-height~=prev-container')
```

Comment:
Malformed string (missing `]`) there are others in this file that can be fixed too.